tests/dhcp_server/test_dhcp_server.py

```diff
@@ -0,0 +1,336 @@
+import binascii
+import logging
+import pytest
+import ptf.packet as scapy
+import ptf.testutils as testutils
+from tests.common.utilities import capture_and_check_packet_on_dut
+from tests.common.helpers.assertions import pytest_assert
+
+
+pytestmark = [
+    pytest.mark.topology('m0', 'mx'),
+]
+
+
+VLAN_GATE_WAY = '192.168.0.1'
```

Comment:
Could vlan gateway and subnet mask be fetched by running config rather than const? It would be more flexible
duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']

Reply:
Yes, added code to parse VLAN from running config instead of const.